test/com/adgoji/mollie/api_test.clj

```diff
@@ -1,7 +1,7 @@
 (ns com.adgoji.mollie.api-test
   (:require
-   [clojure.edn :as edn]
-   [clojure.test :refer [deftest is testing]]
+   [clojure.test :refer [deftest is testing use-fixtures]]
+   [com.adgoji.mollie.utils :refer [with-mollie-client *mollie-client* default-metadata api-key profile-id]]
```

Comment:
I prefer to use namespace aliases (with a few exceptions, such as from `clojure.test`). Let's change it to `[com.adgoji.mollie.utils :as utils]` or even `mollie.utils`. Here is a good article on the topic https://stuartsierra.com/2015/05/10/clojure-namespace-aliases.